share/module_check_a_mundo.F

```diff
@@ -2229,6 +2229,22 @@ END FUNCTION bep_bem_nbui_max
       END IF
 #endif
 
+!-----------------------------------------------------------------------
+!  If the RRTMG KOREA schemes are requested, check that the code with
+!  built to use them.
+!-----------------------------------------------------------------------
+
+#if( BUILD_RRTMK != 1)
+      IF ( ( model_config_rec % ra_lw_physics(1) .EQ. RRTMK_LWSCHEME_FAST )  .OR. &
+           ( model_config_rec % ra_sw_physics(1) .EQ. RRTMK_SWSCHEME_FAST )  ) THEN
```

Comment:
@davegill A pair of eagle eyes also found that it should just be RRTMK_LWSCHEME and RRTMK_SWSCHEME, not _FAST at the end..

Reply:
Good catch, done
🦅 👁 👍 🥇 🎉